templates/contribute/style.md

```diff
@@ -80,7 +80,8 @@ two main concepts in the theory of xyzzyology.
 ## Main results
 
 - `exists_foo`: the main existence theorem of `foo`s.
-- `bar_of_foo`: a construction of a `bar`, given a `foo`.
+- `bar_of_foo`: a construction of a `bar`, given a `foo` under very complicated
+  and intricate circumstances worth describing in excruciating detail.
```

Comment:
I don’t understand what this change is doing here.

Reply:
The change is meant to illustrate an example, where an item spans multiple lines. (I got this wrong previously, by not indenting the second line: this is not how markdown expects items. Hence, I think this is worth clarifying.)

It seems the wording is not clear; I'm very happy to change it. For instance, would this be clearer?
```
- `bar_of_foo_of_baz`: a construction of a `bar`, given a `foo` and a `baz`. If this doc-string was longer than one
  line, subsequent lines should be indented by two spaces (as required by markdown syntax).

Reply:
I have updated the phrasing. @PatrickMassot Does that address your comment?